### PR TITLE
Update README.md to include core principles

### PR DESCRIPTION
**Pull request by hikalium**
Commit message body:
```
Add a section that describes about core principles of lium to help contributors understand our goal clearly.
Also, updated the contribution section to include some additional expectations for the pull requests.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,27 @@
-# lium - Abstraction Layer of ChromiumOS development
+# lium - Make ChromiumOS development extreamely easy
 
-`lium` is an abstruction layer of ChromiumOS development environment and workflows.
+`lium` is an extreamely user-friendly tool for ChromiumOS developers.
 
-It provides a simple interface for common tasks for ChromiumOS developers,
-with plenty of automatic error recovery mechanisms to avoid keep sticking your eyes on the display(s),
-give you some time for a nap and/or coffee, or other tasks ;)
+It provides a simple way to do common development tasks and make the bareer for contributing to ChromiumOS and lium itself low as much as possible.
```
Comment (brandonpollack23) [inline README.md:5]: Nit: barrier
Itself
As low as possible

File: README.md [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+
+It also comes with plenty of automatic error recovery mechanisms to avoid keep sticking your eyes on displays.
```
Comment (brandonpollack23) [inline README.md:7]: I don't understand keep sticking your eyes on displays

File: README.md [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
 
 Also, it manages local development hardware including DUTs and Servos, and act as working examples of commands to interact with them.
 
+We hope lium gives you some time for a nap and/or coffee, or other tasks by making your work more effective ;)
+
+## Core principles
+
+- P0: Make the basic ChromiumOS development workflow extremely easy as like as a usual software on GitHub
```
Comment (brandonpollack23) [inline README.md:15]: Make... extremely easy. The goal is to lower the complexity and bar of entry for contribution. It should be as easy to  pull request into lium as any other project on GitHub.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,27 @@
...
+  - Background: There have been a high barrier to get started with the ChromiumOS development. It terrifies new comers sometimes, and such environment is not sustainable nor efficient. This tool aims to solve such problems at a top priority.
```
Comment (brandonpollack23) [inline README.md:16]: Have been huge barriers
Newcomers
Is not sustainable or efficient
The top priority of lium is to mitigate this high barrier. 

Idk why but this just sounds better to me. It softens it

**Review by brandonpollack23 — COMMENTED** [review]
Quick first pass. 

I'll finish when I get into work today

I'll also draft a section on stacked diffs and why we've decided not to go with them for now but you're encouraged to make each commit insolatable in a PR so if it needs to be reverted fixing the issue is easy.

**Comment by brandonpollack23** [comment]
Stacked Diffs:

We discussed using a tool like git-ps-rs or ghstack, and while we believe stacked commits as they work on our internal tools are best, we decided the priority of lowering the bar of entry for contribution was more important.  That being said, we do reccomend making each commit hermetic and before seinding a pull request we request you run:
```
git rebase --exec "make check"
```

If this works it means each commit is safe and can be reverted independently from the following commits.

If you do this, for now you can let us know squashing isn't necessary and that each commit is hermetic.  In the future we'll evaluate if this policy is working out or not, and adjust accordingly.

File: README.md (hunk 4 above)
Comment (cynthia) [reply README.md:16]: JFYI: P0 is a Googleism, I would spell it out as top priority.

File: README.md (hunk 2 above)
Comment (hikalium) [reply README.md:7]: removed the line

File: README.md (hunk 4 above)
Comment (hikalium) [reply README.md]: I think P0 term is now commonly used so I would like to keep it as is.
updated the text based on Brandon's suggestion

**Review by mhiramat — APPROVED** [review]
LGTM!

File: README.md (hunk 4 above)
Comment (cynthia) [reply README.md]: > P0 – the site is down and all work stops, until this issue is resolved.

https://www.linkedin.com/pulse/definition-priority-codes-p0-p1-p2-p3-p4-technical-victor-pudeyev

Also: https://en.m.wikipedia.org/wiki/P0

has nothing.

File: README.md [hunk 5]
```diff
@@ -1,13 +1,27 @@
...
+  - Basic workflows include: checkout the source code, build images / deploying packages / run tests with / without modifications.
+- P1: Make it extreamely easy to start using lium
+  - Background: As stated in the above, our goal is lowering the barrier for people who are about to start contribution to ChromiumOS. To achive that, a tool that aids the goal should be extremely easy as well to start using it.
+  - How: Follow best practices and common ways to do things. Prefer defaults always if there is no clear reason to change that.
+- P2: Be a working example compiled from knowledge in the documentations
```
[the remark this answers is not in the document]
Comment (brandonpollack23) [reply README.md:19]: "It terrifies" might be a little harsh for a google hosted doc...maybe "It indimidates"

...is not sustainable, efficient, or scalable

mitigate this high barrier of entry

File: README.md [hunk 6]
```diff
@@ -1,13 +1,25 @@
...
+
+## Core principles
+
+- P0: Make the basic ChromiumOS development workflow extremely easy
+  - Background: There have been a huge barriers to get started with the ChromiumOS development. It terrifies newcomers sometimes, and such environment is not sustainable or efficient. The top priority of lium is to mitigate this high barrier to enter.
+  - Basic workflows include: checkout the source code, build images / deploying packages / run tests with / without modifications.
+- P1: Make it extreamely easy to start using lium
+  - Background: As stated in the above, our goal is lowering the barrier for people who are about to start contribution to ChromiumOS. To achive that, a tool that aids the goal should be extremely easy as well to start using it.
+  - How: Follow best practices and common ways to do things. Prefer defaults always if there is no clear reason to change that.
+- P2: Be a working example compiled from knowledge in the documentations
```
Comment (brandonpollack23) [inline README.md:19]: nit: knowledge in the documentation

File: README.md [hunk 7]
```diff
@@ -1,13 +1,25 @@
...
+  - Background: Documentations can be rotten sirently. Code rots as well, but it will stop working sooner. People often prefer coding than documentation.
```
Comment (brandonpollack23) [inline README.md:20]: coding over documentation

File: README.md [hunk 8]
```diff
@@ -1,13 +1,25 @@
...
+  - How: Put enough background information in the code as comments. Put links to the documentations. Put anything useful with code, and wrap them with logics. It gives better understanding on the things than only using natural languages.
```
Comment (brandonpollack23) [inline README.md:21]: ditto, documentation is singular

Comment (brandonpollack23) [inline README.md:21]: nit: Consistent subject

Either change Put enough to It puts enough or change It gives to Give

Comment (brandonpollack23) [inline README.md:21]: Can you rephrase, I dont quite understand "on the things thatn only using natural languages"

**Review by brandonpollack23 — COMMENTED** [review]
Lots of grammar nits, but thanks for all the clarifications

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (hikalium) [reply README.md:19]: done!
